tools/setup/emoji/build_emoji

```diff
@@ -153,6 +155,9 @@ def code_point_to_file_name_map(ttx):
 
 def main():
     # type: () -> None
+    run(["sudo", "mkdir", "-p", EMOJI_CACHE_PATH])
+    run(["sudo", "chown", "%s:%s" % (user_id, user_id), EMOJI_CACHE_PATH])
```

Comment:
Hmm, does this `sudo` work?

I don't see an existing `sudo` call in this file. And in the places this script is called that are mentioned elsewhere in this commit, the `mkdir`/`chown` are under `sudo` but the call to this script isn't.

Oh, in fact, searching for where `EMOJI_CACHE_PATH` is defined here, there's this:
```
if 'TRAVIS' in os.environ:
    # In Travis CI, we don't have root access
    EMOJI_CACHE_PATH = "/home/travis/zulip-emoji-cache"
```
So it sounds like this may not work in Travis.